docs/aop/user/apps/bazzarvoice.md

```diff
@@ -0,0 +1,85 @@
+template: howto-guide-template
+---
+
+The [BazzarVoice](https://www.bazaarvoice.com/) app lets you collect and add user-generated content (UGC) to your product pages. 
+
+The BazzarVoice service offers the following UGC: 
+
+- [Rating summaries](https://knowledge.bazaarvoice.com/wp-content/conversations/en_US/Display/display_integration.html#rating-summary), or star ratings
+- [Product reviews](https://knowledge.bazaarvoice.com/wp-content/conversations/en_US/Display/display_integration.html#reviews)  
+<!---- [Questions and answers](https://knowledge.bazaarvoice.com/wp-content/conversations/en_US/Display/display_integration.html#questions--answers)-->
+
+BazzarVoice uses the content syndication approach, which means that stores using BazzarVoice republish each others' content. For example, if a store within the BazzarVoice's network has got a new product review, this review is shared across all other stores in the network that also have this product.
+
+{% info_block warningBox "Important" %}
+
+To enable BazzarVoice to match your products to products in other stores and upload product reviews into your store, you must use UPCs or EANs as unique identifiers for your products.
+
+{% endinfo_block %}
+
+When you connect BazzarVoice, the app puts JavaScrip tags into your store, and the JavaScript code tells the app where to insert the BazzarVoice content - reviews, star ratings, or questions and answers.
+
+{% info_block infoBox "Info" %}
+
+If you have BazzarVoice integrated, the Spryker default [Product Ratings and Reviews feature](/docs/scos/user/features/{{page.version}}/product-rating-and-reviews-feature-overview.html) is turned off. This means that ratings and reviews collected with the default Spryker Product Ratings and Reviews feature are replaced with the BazzareVoice ratings and reviews.
+
+{% endinfo_block %}
+
+## BazzarVoice integration and configuration
+
+To integrate or configure BazzarVoice:
+
+1. In your store's Back Office, go to *Apps*.
+2. Click BazzarVoice.
+   This takes you to the BazzarVoice app details page, from where you can do the integration and configuration.
+
+### Integrating BazzarVoice into your store
+
+To integrate the BazzarVoice app into your store:
+
+1. In the top right corner of the BazzarVoice app details page, click **Connect app**.
+   This takes you to the BazzarVoice site with the signup form.
+2. Fill out the BazzarVoice signup form and submit it.
+   You should receive the BazzarVoice credentials.
+3. Go back to your store's Back Office, to the BazzarVoice app details page.
+4. In the top right corner of the BazzarVoice app details page, click **Configure**.
+5. In the *Configure* pane, enter the credentials you received from BazzarVoice.
+
+That's it. You have integrated the BazzarVoice app into your store. It usually takes BazzarVoice a few days to process your product feed. Therefore, you should see the external ratings and reviews from BazzarVoice in about 2-3 days after you integrated the app.
```

Comment:
(There will be some new content for this line, as we are changing from a product feed approach to a dynamic catalog collection approach. Will give you new info ASAP.)